### PR TITLE
Follow up PR

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

Follow up PR

### CODE REVIEW

File: jazz_create-serverless-service/index.js [hunk 1]
```diff
@@ -119,8 +114,7 @@ module.exports.handler = (event, context, cb) => {
                         reject(err);
                     } else {
                         if (response.statusCode <= 299) { // handle all 2xx response codes as success
-                            var messageToBeSent = "Your service code will be available at " + config.BIT_BUCKET_URL + bitbucketName + "/browse";
-                            resolve(messageToBeSent);
+                            resolve("Successfully created your service.");
```
Comment (suryajak) [inline jazz_create-serverless-service/index.js]: Remove BIT_BUCKET_URL from config file(s)?

File: jazz_create-serverless-service/index.js [hunk 2]
```diff
@@ -93,10 +92,6 @@ module.exports.handler = (event, context, cb) => {
                 var domain = (event.body.domain || "").toLowerCase();
                 var service_name = event.body.service_name.toLowerCase();
 
-                var bitbucketName = service_name;
```
Comment (suryajak) [inline jazz_create-serverless-service/index.js]: Call this function startServiceOnboarding

Comment (SiniWilson) [reply jazz_create-serverless-service/index.js]: In cooperated the change

File: jazz_create-serverless-service/index.js (hunk 1 above)
Comment (SiniWilson) [reply jazz_create-serverless-service/index.js]: In cooperated the change

**Pull request merged**